react/src/lib/components/DeckGLMap/layers/LassoLayer/lassoLayer.stories.tsx

```diff
@@ -0,0 +1,124 @@
+import { makeStyles } from "@material-ui/core";
+import Switch from "@material-ui/core/Switch";
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import React from "react";
+import DeckGLMap from "../../DeckGLMap";
+
+export default {
+    component: DeckGLMap,
+    title: "DeckGLMap / Lasso Layer",
+} as ComponentMeta<typeof DeckGLMap>;
+
+const useStyles = makeStyles({
+    main: {
+        height: 500,
+        border: "1px solid black",
+        position: "relative",
+    },
+    legend: {
+        width: 100,
+        position: "absolute",
+        top: "0",
+        right: "0",
+    },
+});
+
+export const lassoLayerTemplate: ComponentStory<typeof DeckGLMap> = (args) => {
+    const [editedData, setEditedData] = React.useState(args.editedData);
+    const [argsState, setArgsState] =
+        React.useState<Record<string, unknown>>(enableLassoArgs);
+    const [state, setState] = React.useState<boolean>(true);
+
+    const handleChange = React.useCallback(() => {
+        const lassoLayer = enableLassoArgs.layers.filter(
+            (item) => item["@@type"] === "LassoLayer"
+        );
+        if (lassoLayer[0].visible !== undefined) {
+            lassoLayer[0].visible = !lassoLayer[0].visible;
+        }
+        if (lassoLayer[0].visible) {
+            setArgsState(enableLassoArgs);
+        } else {
+            setArgsState(disableLassoArgs);
+        }
+        setState(!state);
+    }, [state]);
+
+    React.useEffect(() => {
+        setEditedData(args.editedData);
+    }, [args.editedData]);
+
+    return (
+        <>
+            <div className={useStyles().main}>
+                <DeckGLMap
+                    id={"DeckGL-Map"}
+                    {...argsState}
+                    editedData={editedData}
+                    setProps={(updatedProps) => {
+                        setEditedData(updatedProps);
+                    }}
+                    legend={{ visible: false }}
+                />
+            </div>
+            <div style={{ textAlign: "center" }}>
+                <div style={{ display: "inline" }}>Display lasso layer</div>
```

Comment:
Suggest using a Label instead, in order to increase the clicking surface: https://mui.com/material-ui/react-switch/#label